srcs/textures.c

```diff
@@ -0,0 +1,24 @@
+#include "cub.h"
+
+void	init_textures(t_cub *cub)
```

Comment:
I added some stuff to init_mlx myself, and actually changed the name of the function to init_mlx_and_raycast. I'm honestly not super happy about my own choice, and I think we need to rethink a bit how we initialize stuff. I'd say that it makes a lot of sense to call this init_textures from inside init_mlx, but we need it as a separate function for norminette purposes. Still I'd move this to the initializers file (I guess at some point we'll need an initializers folder aha)

Reply:
I moved it to initializers for now.